sops/european-level/GDI-SOP0009_dataset-withdrawal.md

```diff
@@ -0,0 +1,466 @@
+- **Review** ``dataset-withdrawal-request`` and ``dataset-withdrawal`` ticket details.
+- **Appoint a Technical Lead** (TL) per GDI system (may be one or several people).
+   - To know better who is responsible for what system, you should:
+      - Request help from Pillar II Work Package (WP) leads and OC/SDPC.
+      - Take a look at GDI nodes' service portfolios. Also see hierarchy of GDI systems and relevant SOPs at [Step 7](#87-per-system-dataset-withdrawal).
+   - TL **responsibilities** (aided by respective SOPs at [Step 7](#87-per-system-dataset-withdrawal)):  
+     - Identify dataset presence (full or partial) in their respective GDI system.
+     - Remove data if present.
+   - **Send task email** to each appointed TL:
+````
+To: 
+CC: gdi-sdpc [at] elixir-europe.org, gdi-oc [at] elixir-europe.org
+Subject: [<dataset-withdrawal TICKET_ID>][GDI SOP0009] Dataset withdrawal for <GDI_SYSTEM>
+````
+````
+Dear <TL_NAME>,
+
+You are appointed **Technical Lead** (TL) for the dataset-withdrawal request detailed below:
+
+- Ticket ID: <dataset-withdrawal TICKET_ID from GDI's VHD> (<TICKET_URL if applicable>)
+- Dataset IDs: <list of relevant dataset IDs>
+- Scope: <Whether the dataset is to be fully or partially withdrawn>
+- System: <GDI system's name>
+- Stated reason: <Reason for withdrawal from the requester>
+- Requester authority: <copy-pasted from previous VHD officer communication>
+- Target completion date: <Deadline of withdrawal if applicable>
+
+ACTIONS NEEDED:
+1. Acknowledge receipt of this email **within 3 working days**.
+2. Determining whether any (meta)data of the dataset resides in the specified GDI system.
+3. If present, remove it (within the given scope) as per system SOP (see [Step 7](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/sops/european-level/GDI-SOP0009_dataset-withdrawal.md#87-per-system-dataset-withdrawal)).
+4. Report dataset withdrawal status in the ticket or directly contacting the GDI VHD.
+
+You can find more details about how these tasks have been fulfilled previously at [Step 7](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/sops/european-level/GDI-SOP0009_dataset-withdrawal.md#87-per-system-dataset-withdrawal).
+
+If you need additional context or support, contact the OC/SDPC (CC'ed) or the VHD officer (<VHD_OFFICER_CONTACT_DETAILS>).
+Thank you for your prompt attention.
+
+Best regards,
+<YOUR_NAME>
+<YOUR_ROLE>
+<YOUR_INSTITUTION>
+OC/SDPC Delegate
+Genomic Data Infrastructure (GDI)
+````
+- Once a TL has been assigned (i.e., each has acknowledged it) to each of the GDI systems, **record assignments and target completion date** in the ``dataset-withdrawal`` ticket.
+- **Notify VHD officer** (who managed the initial request) that leads are assigned. Include in the communication _who_ has been assigned _what_.
+- This **concludes** your responsibilities as the OC/SDPC delegate with respect to this SOP. 🔚
```

Comment:
As mentioned above, this should be an automatic process when the node withdraws the data.